src/http.rs

```diff
@@ -36,10 +43,15 @@ where
         Ok(HTTP {
             client: C::default(),
             app_name,
+            sdk_version: get_sdk_version(),
+            connection_id: Uuid::new_v4().into(),
             instance_id,
             authorization,
             authorization_header: C::build_header("authorization")?,
             app_name_header: C::build_header("appname")?,
```

Comment:
this header will be deprecated in the next version since we're moving towards standardized `x-unleash` headers

Reply:
I haven't seen these headers before. Are they already in production somewhere?

Reply:
If not, please see https://datatracker.ietf.org/doc/html/rfc6648 - x- is a deprecated pattern and we SHOULD NOT introduce it now.

Reply:
We started adding them 2 weeks ago for our internal purposes. Those are not useful to Unleash users and only used internally so we have more freedom in changing them in future. You raise a good point with the x-prefix. I'll discuss it with the team tomorrow (if it's worth changing it given all other SDKs already support it)

Reply:
We'll update our convention and drop the x-prefix and call it unleash-....

Reply:
Update: after internal discussions we decided to drop the x- prefix and we'll stick to unleash-. The spec link you sent is convincing :)